backend/api/Areas/Project/Controllers/SearchController.cs

```diff
@@ -58,7 +58,7 @@ public IActionResult GetProjectsPage()
         {
             var uri = new Uri(this.Request.GetDisplayUrl());
             var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
-            return GetProjectsPage(new ProjectFilterModel(query));
+            return GetProjectsPage(new ProjectFilter(query));
```

Comment:
Do we need the `ProjectFilterModel` at all?  Can it be deleted if we just use the `ProjectFilter` instead?

Reply:
Not needed anymore